app/geo/gui/src/ts/twxplore/gui/geo/reducers/map/mapReducer.ts

```diff
@@ -45,47 +45,34 @@ export const mapReducer = (state: MapState, action: BaseAction): MapState => {
           }
         }
         /*Need to implement how to do this more dynamically. For now this works*/
-        if (result.featureTypesFilters[addedFeature.type!]) {
-          if (addedFeature.timestamp) {
-            if (
-              addedFeature.timestamp <
-              result.featureTypesFilters[addedFeature.type!].timestamp.min!
-            )
-              result.featureTypesFilters[addedFeature.type!].timestamp.min =
-                addedFeature.timestamp;
-            else if (
-              addedFeature.timestamp >
-              result.featureTypesFilters[addedFeature.type!].timestamp.max!
-            )
-              result.featureTypesFilters[addedFeature.type!].timestamp.max =
-                addedFeature.timestamp;
-          }
+        if (addedFeature.timestamp) {
+          if (
+            addedFeature.timestamp <
+            result.featureTypesFilters[addedFeature.type!].timestamp.min!
```

Comment:
Pull
result.featureTypesFilters[addedFeature.type!]
into a variable, please, this code is hard to read.